index.html

```diff
@@ -4,272 +4,289 @@
     <meta charset="UTF-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <link rel="stylesheet" href="resources/css/reset.css" />
-    <link rel="stylesheet" href="resources/css/style.css" />
+    <link rel="stylesheet" href="/resources/css/reset.css" />
+    <link rel="stylesheet" href="/resources/css/style.css" />
     <title>Portfolio</title>
   </head>
   <body>
-    <main>
-      <header class="top-nav-burger">
-        <div class="top-corner-image">
-          <img
-            src="/resources/images/image_geometry_1.png"
-            alt="corner image"
-          />
```

Comment:
[ Optional Changes ]

- [ ] Kindly work on all images path in your project. All your images seem to not rendering and are broken. You can you the format below to trace the path of your images `src="./resources/images/image_geometry_1.png"`
- [ ] Please check all your images and fix that issue. Also, please don't forget the CSS, you used images there as well. 👍